src/lib/inputs/input.scss

```diff
@@ -8,8 +8,8 @@
     input:not([type='color']) {
       box-sizing: content-box;
       height: 28px;
-      padding-top: 11px;
```

Comment:
What was this change about? That will affect all our inputs right?

Reply:
I adjusted this, because the padding was off in storybook.